lib/src/disposable/transformer/transformations.dart

```diff
@@ -0,0 +1,189 @@
+// Copyright 2016 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:analyzer/analyzer.dart';
+import 'package:transformer_utils/transformer_utils.dart';
+
+const String _generatedPrefix = r'_$';
+const String _publicGeneratedPrefix = r'$';
```

Comment:
Is it possible for `$` prefix to clash with other transformers? It might be a safe assumption that `${ClassName}NullOutFields` won't be taken, but I know I've already implemented a similar such method in `w_comments` and almost did go with a very similar name.